openslides/core/views.py

```diff
@@ -46,6 +56,192 @@ def get(self, *args, **kwargs):
         return HttpResponse(content)
 
 
+class ProjectorViewSet(ReadOnlyModelViewSet):
+    """
+    API endpoint to list, retrieve and update the projector slide info.
+    """
+    queryset = Projector.objects.all()
+    serializer_class = ProjectorSerializer
+
+    def check_permissions(self, request):
+        """
+        Calls self.permission_denied() if the requesting user has not the
+        permission to see the projector and in case of an update request the
+        permission to manage the projector.
+        """
+        manage_methods = (
+            'activate_projector_elements',
+            'clear_and_activate_projector_elements',
+            'deactivate_projector_elements',
+            'deactivate_all_projector_elements')
+        if (not request.user.has_perm('core.can_see_projector') or
+                (self.action in manage_methods and
+                 not request.user.has_perm('core.can_manage_projector'))):
+            self.permission_denied(request)
+
+    @detail_route(methods=['post'])
+    def activate_projector_elements(self, request, pk):
```

Comment:
Please rename to 'activate' or at lease 'activate_elements'. So the uri is not /rest/core/projector/1/activate_projector_elements/